release/release_gold.sh

```diff
@@ -149,7 +149,7 @@ function prepare_next_release {
 		rm --force --recursive releases.json
 	fi
 
-	if [ "${_PRODUCT_VERSION}" != "${latest_quarterly_product_version}" ]
+	if [ "${_PRODUCT_VERSION%-lts}" != "${latest_quarterly_product_version}" ]
```

Comment:
@lucasmiranda0, we already have a function which does exactly what you're looking for, please see: https://github.com/liferay/liferay-docker/blob/05b5136b24570e7bd646e557c707023a3862776e/_release_common.sh#L117-L122.

Can you use this function here?